google_guest_agent/getrelease_integ_test.go

```diff
@@ -0,0 +1,18 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"testing"
+)
+
+func TestGetRelease(t *testing.T) {
+	osrelease := getRelease()
+	if osrelease.os == "" {
+		t.Errorf("failed to get os name")
+	}
+	if osrelease.version.String() == "" {
+		t.Errorf("failed to get os version")
```

Comment:
any idea for checking that it worked successfully?

Reply:
My idea is that we could compare the what get release returned with metadata server. I didn't